components/ArrowPath/arrow.ts

```diff
@@ -0,0 +1,136 @@
+import {Layer} from '@deck.gl/core';
+// @ts-ignore
```

Comment:
If you need to use a ts-ignore, better to ignore a specific error and write a comment as to why this is turned off. Typescript might seem like a pain but in the long run the type checking is super useful